src/cpp/rtps/transport/ChainingReceiverResource.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef TRANSPORT_CHAININGRECEIVERRESOURCE_HPP
```

Comment:
I am also missing the copyright and license notice

Reply:
Done